dvc/command/experiments.py

```diff
@@ -848,6 +910,89 @@ def add_parser(subparsers, parent_parser):
     )
     experiments_branch_parser.set_defaults(func=CmdExperimentsBranch)
 
+    EXPERIMENTS_LIST_HELP = "List local and remote experiments."
+    experiments_list_parser = experiments_subparsers.add_parser(
+        "list",
+        parents=[parent_parser],
+        description=append_doc_link(EXPERIMENTS_LIST_HELP, "experiments/list"),
+        help=EXPERIMENTS_LIST_HELP,
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+    )
+    experiments_list_parser.add_argument(
+        "-r",
```

Comment:
About short options. They overlap with some existing ones (--remote, --all-branches, --force), so maybe let's start with long ones for now?